torch/csrc/jit/passes/subgraph_rewrite.h

```diff
@@ -47,7 +48,16 @@ class TORCH_API SubgraphRewriter {
   script::Module runOnModule(const script::Module& module);
 
   // Run pattern-based subgraph rewrite pass on the graph (used in testing).
-  void runOnGraph(std::shared_ptr<Graph>& graph);
+  // filter is a function that does extra filtering on the match, if it returns
+  // true for a given Match, we'll skip the match
```

Comment:
The comment contradicts with the code: we skip if the filter returns false.

Also, please add a detailed description of the arguments of this function - what is the meaning of its arguments and how in a usual case users of this API could obtain them.